well-sample/src/main/java/com/yarolegovich/wellsample/WellConfig.java

```diff
@@ -70,6 +70,11 @@ public String getDbName() {
         return "my_db";
     }
 
+    @Override
+    public long getCursorWindowSize() {
```

Comment:
Should we move this to `DefaultWellConfig` and use the default (2MB) value, then we could do whatever on our side in FluxC (inject the right config when in a debug variant or not)?

Reply:
I considered that, but thought it best to make sure we only set the cursor window size on purpose. There's no guarantee that Android will always have the current 2MB limit.

Reply:
Good point, I wanted to make sure we had an default or abstract version of this function - and I just noticed there was an `interface WellConfig` so this is fine!